### PR TITLE
closing conference on Android

### LINKED ISSUE #39: Can we close Jitsi meeting room programatically?
Hi, 
How can we stop the meeting call pragmatically on an event.

### LINKED ISSUE #45: Could you please implement an end meeting functionality from the dart code ?
In my app, there is a functionality to end the meeting programmatically when the host stopped the meeting. Please check about this https://jitsi.github.io/handbook/docs/dev-guide/dev-guide-android-sdk#leave

### PR DESCRIPTION
**Pull request by niyaz1998**
I created function to close a conference from Flutter code. Works only for Android, same function for IOS will be developed later. Please provide feedback, if something is wrong.

### CODE REVIEW

File: android/src/main/kotlin/com/gunschu/jitsi_meet/JitsiMeetPluginActivity.kt [hunk 1]
```diff
@@ -26,27 +29,33 @@ class JitsiMeetPluginActivity: JitsiMeetActivity() {
...
             getJitsiView().leave()
         }
     }
 
-    override fun onStop()
-    {
+    private val myReceiver: BroadcastReceiver = object : BroadcastReceiver() {
+        override fun onReceive(context: Context, intent: Intent) {
+            when (intent?.action) {
+                JITSI_MEETING_CLOSE -> finish()
```
Comment (tuantvu) [inline android/src/main/kotlin/com/gunschu/jitsi_meet/JitsiMeetPluginActivity.kt:44]: Have you tried getJitsiView().leave() as above? Trying to keep consistency as well as maybe there is some clean up that leave() call does we don't want to miss out on.

**Comment by tuantvu** [comment]
@niyaz1998 Great work. Made one comment on the code. Also, have you started working on IOS version yet? I want to make sure there is a solution for it and we support both platforms. Just a short explanation of how it could work is okay too.

**Comment by niyaz1998** [comment]
@tuantvu I asked my college to do this functionality for IOS. We will update Pull Request soon.

**Comment by tuantvu** [comment]
@niyaz1998 Good to hear. Do you want me to merge this pull now and submit a different one later? Also, do you think there is anything we can add to the example app to show case how to use this functionality?

**Comment by invitro** [comment]
any update about this @niyaz1998 ?

**Comment by niyaz1998** [comment]
other tasks are in priority, so I can't tell anything right now

**Comment by invitro** [comment]
@tuantvu can we merge this pull request first for a while and update with a new version from @niyaz1998 later? because I believe some of developers need this features. Thanks

**Comment by niyaz1998** [comment]
Yes, I think, but it works only on Android, better to add comment

**Comment by invitro** [comment]
Thanks @tuantvu for the release and I (and some developers) hope you can have time soon to finish the ios version soon @niyaz1998 ... thanks anyway

**Comment by jimmyjose-dev** [comment]
> Thanks @tuantvu for the release and I (and some developers) hope you can have time soon to finish the ios version soon @niyaz1998 ... thanks anyway

#73 Close meeting programmatically in iOS

**Pull request merged**